style/properties/shorthands/text.mako.rs

```diff
@@ -47,6 +51,11 @@
                 parse_component!(thickness, text_decoration_thickness);
             % endif
 
+            % if engine == "servo":
+                parse_component!(style, text_decoration_style);
+                parse_component!(color, text_decoration_color);
```

Comment:
Take common logic out of the conditionals.

Reply:
done